runelite-client/src/main/java/net/runelite/client/ui/overlay/OverlayRenderer.java

```diff
@@ -506,11 +507,35 @@ private void safeRender(Client client, Overlay overlay, OverlayLayer layer, Grap
 		final OverlayPosition position = overlay.getPosition();
 
 		// Set font based on configuration
+		if ((position == OverlayPosition.DYNAMIC || position == OverlayPosition.TOOLTIP))
```

Comment:
probably do this only for DYNAMIC overlays, and always set small font for tooltips.